programs/bpf_loader/src/syscalls.rs

```diff
@@ -1471,7 +1471,7 @@ fn check_instruction_size(
     data_len: usize,
     invoke_context: &Ref<&mut dyn InvokeContext>,
 ) -> Result<(), EbpfError<BPFError>> {
-    let size = num_accounts * size_of::<AccountMeta>() + data_len;
+    let size = num_accounts.saturating_mul(size_of::<AccountMeta>().saturating_add(data_len));
```

Comment:
Order of operations:
```suggestion
    let size = num_accounts.saturating_mul(size_of::<AccountMeta>()).saturating_add(data_len);
```

Reply:
Whoops, thanks! :slightly_smiling_face: